Add JWT Verify, Decode and Sign Operators

GCHQ77703 opened the pull request:
Having some trouble designing tests for these, due to the creation of an `iat` token within the JSON response that represents the time that the token was created at.  Ideas are welcome.

n1474335 commented:
You probably want to be using an `option` argument rather than a `populateOption`. See my comment in #347.

GCHQ77703 commented:
Fixed the tests, found I can just force the `iat` value to whatever I want.  Due to library limitations, it appears that the verification of `ES` and `RS` signature schemes is broken so they have been disabled.  Decoding / encoding to those types still works.

Also fixed the `option` issue, thanks for helping out!

n1474335 commented:
Awesome work, thanks very much. I made a few changes to improve error handling and the descriptions.
